### ISSUE TITLE
Fixing the dates in the 2018 data reveals a new bugs in the code

### ISSUE THREAD

**Issue by davidjpmoore** [davidjpmoore/Fsoil_aridlands#10]
I think that @AnastasiaAVM needs to look at this file now. I think I fixed the dates but it reveals another error later on - 

The SWC values are much to high when I run it. It would be worth checking this - I cannot track down where the error is.

**Comment by AnastasiaAVM** [comment]
Yes, we have some problems here with all three depths of SWC - seems like there was some problem with moisture sensor or during again "gap filling procedure" - some data has been changed.
Probably concerning this issue, we also need to contact Russ.
